docs/topics/Permissions.md

```diff
@@ -26,48 +26,50 @@ Below is a table of all current permissions, their integer values in hexadecimal
+| MANAGE_WEBHOOKS \*            | `0x0000000020000000` `(1L << 29)` | Allows management and editing of webhooks                                                                                                                   | T            |
+| MANAGE_EMOJIS_AND_STICKERS \* | `0x0000000040000000` `(1L << 30)` | Allows management and editing of emojis and stickers                                                                                                        |              |
+| USE_APPLICATION_COMMANDS      | `0x0000000080000000` `(1L << 31)` | Allows members to use application commands, including slash commands and context menu commands.                                                             | T            |
+| REQUEST_TO_SPEAK              | `0x0000000100000000` `(1L << 32)` | Allows for requesting to speak in stage channels. (_This permission is under active development and may be changed or removed._)                            | S            |
+ | MANAGE_EVENTS                | `0x0000000200000000` `(1L << 33)` | Allows for creating, editing, and deleting scheduled events                                                                                                 | V, S         |
+| MANAGE_THREADS \*             | `0x0000000400000000` `(1L << 34)` | Allows for deleting and archiving threads, and viewing all private threads                                                                                  | T            |
+| CREATE_PUBLIC_THREADS         | `0x0000000800000000` `(1L << 35)` | Allows for creating threads                                                                                                                                 | T            |
+| CREATE_PRIVATE_THREADS        | `0x0000001000000000` `(1L << 36)` | Allows for creating private threads                                                                                                                         | T            |
+| USE_EXTERNAL_STICKERS         | `0x0000002000000000` `(1L << 37)` | Allows the usage of custom stickers from other servers                                                                                                      | T            |
+| SEND_MESSAGES_IN_THREADS      | `0x0000004000000000` `(1L << 38)` | Allows for sending messages in threads                                                                                                                      | T            |
+| START_EMBEDDED_ACTIVITIES     | `0x0000008000000000` `(1L << 39)` | Allows for launching activities (applications with the `EMBEDDED` flag) in a voice channel                                                                  | V            |
+| MODERATE_MEMBERS              | `0x0000010000000000` `(1L << 40)` | When you put a user in timeout they will not be able to send messages in chat, reply within threads, react to messages, or speak in voice or stage channels |              |
```

Comment:
looks good! NIT: would be good to keep description a bit more consistent with others, maybe something like:
"Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels".

Reply:
Sure!